03-Node-npm-modules/js/lib/jquery.js

```diff
@@ -0,0 +1,9 @@
+$ = require('jquery');
```

Comment:
This is wrong, the exercise is only asking to import jquery not to do this. remove this file and use this statement on the Movie file.